macros/models/_project/balancer/balancer_liquidity_macro.sql

```diff
@@ -31,6 +31,9 @@ WITH pool_labels AS (
             approx_percentile(median_price, 0.5) AS price,
             sum(sample_size) AS sample_size
         FROM {{ source('dex', 'prices') }}
+        WHERE 
+        ('{{blockchain}}' != 'fantom' OR 
+        ('{{blockchain}}' = 'fantom' AND contract_address NOT IN (0xde1e704dae0b4051e80dabb26ab6ad6c12262da0, 0x5ddb92a5340fd0ead3987d3661afcd6104c3b757))) --broken price feeds
```

Comment:
did we trace these back to the source and see what makes them broken price feeds?
is it broken at the `prices.usd` level, fed by the `prices_fantom.tokens` static spell?
if so, why do these addresses break the `dex.trades` logic to pull price from "trusted token" side of trade, maybe because neither bought/sold have a trusted token?

Reply:
hey, @jeff-dude! good to have you back!
so, both of these tokens have a very low volume of trades, that's why the last time they had their price updated on dex.prices was more than 1 year ago

Reply:
hmm okay. i guess i'm missing the context in terms of how that impacts this spell. as long as it's not a source pricing issue, then i'm good with this